examples/tests/valid/bring_local_dir.test.w

```diff
@@ -1,5 +1,6 @@
 bring "./subdir2/inner/widget.w" as w;
 bring "./subdir2" as subdir;
+bring "./subdir-3" as hasNoWingFiles;
```

Comment:
this should fail, no? (because there are no .w files there)

Reply:
I thought we just want to skip directories with no .w files, rather than error on them

Reply:
if i explicitly try to `bring` a directory that doesn't have any `.w` files i think it should be an error.

in this case, there are two issues:

1. the directory doesn't have any `.w` file.
2. if it had a `.w` file, it would have been an error because the name is invalid.

so by definition This Code Is Bad™️

Reply:
Ah I see where I confused myself. So this will need to be re-worked to only ignore sub directories of the brought directory. Otherwise the brought directory should always contain some `.w` file.

Thanks for setting me straight!

Reply:
yes, this seems like a reasonable direction.